webui/react/src/components/Table/TableFilterDropdown.tsx

```diff
@@ -167,14 +147,29 @@ const TableFilterDropdown: React.FC<Props> = ({
           />
         </div>
       )}
-      <FixedSizeList
-        height={listHeight}
-        itemCount={filteredOptions.length}
-        itemData={filteredOptions}
-        itemSize={ITEM_HEIGHT}
-        width="100%">
-        {OptionRow}
-      </FixedSizeList>
+      <Virtuoso
+        data={filteredOptions}
+        initialItemCount={filteredOptions.length}
+        itemContent={(_index: number, data: ColumnFilterItem) => {
+          const classes = [css.option];
+          const isSelected =
+            typeof data.value !== 'boolean' ? selectedMap[data.value.toString()] : false;
+          const isJSX = typeof data.text !== 'string';
+          if (isSelected) classes.push(css.selected);
+          return (
+            <div
+              className={classes.join(' ')}
+              data-value={data.value}
+              style={{ height: ITEM_HEIGHT }}
```

Comment:
super nit: you could change the padding in the scss file to avoid having to hard code the height